docker-compose.yml

```diff
@@ -0,0 +1,5 @@
+version: '3'
```

Comment:
Yeah, so, I had to make a few decisions while working on this part.

So, `WITH DOCKER` from Earthly cannot easily pull the test Bento server. (I could try and find ways to avoid having a docker-compose file).

And if we would be able to host a local server for testing/make public a repo that I can pull and build within the Earthly run, it would be cool and it will not let us use a docker compose file.

Reply:
It would be perfect if we could do everything with Earthly in order to keep the amount of tools minimum

Reply:
Yeah, I will try searching for ways to go around the Compose file.